dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBanner.tsx

```diff
@@ -517,6 +597,70 @@ const styles = {
 				'.		vert-line	cta-container	${cardsImageOrSpaceTemplateString}	.';
 		}
 	`,
+	collapsedLayoutOverrides: (cardsImageOrSpaceTemplateString: string) => css`
```

Comment:
if the 2-step banner is successful and we keep it then we should try to consolidate this with the css above